Antidote/CallsManager.m

```diff
@@ -140,17 +142,17 @@ - (void) controller:(RBQFetchedResultsController *)controller
             }
             break;
         case NSFetchedResultsChangeDelete: {
+
             if ([self.allCallsController.fetchedObjects count] == 0) {
                 [[AppContext sharedContext] killCallsManager];
                 return;
             }
 
-            if ([self onlyPauseCallsLeft]) {
-                OCTCall *call = [self.allPausedCallsController.fetchedObjects firstObject];
+            if (controller == self.allCallsController) {
 
-                // workaround for deadlock in objcTox
-                // https://github.com/Antidote-for-Tox/objcTox/issues/51
-                [self performSelector:@selector(switchViewControllerForCall:) withObject:call afterDelay:0];
+                if ([anObject.primaryKeyValue isEqualToString:self.currentCall.primaryKeyValue]) {
+                    [self replaceCurrentCallThatWasRemoved];
```

Comment:
Having complete sentence as method name isn't that readable ("Replace current call that was removed"). I think it would be better to call this method `didRemoveCurrentCall`. I think it would be clear enough what this method means. And in future there may be some other logic than replacement the call.